### ISSUE TITLE
Building rustc broken for multiple targets due to lack of `LINK_` entry in cfg/mk/*.mk

### ISSUE THREAD

**Issue by jmesmon** [rust-lang/rust#25802]
Appears like:

```
| CFG_LLVM_LINKAGE_FILE=/home/cody/obj/y/tmp/work/x86_64-poky-linux-gnueabi/rust-cross-arm/git+gitAUTOINC+c654a07d29-r0/build/x86_64-linux/rt/llvmdeps.rs LD_LIBRARY_PATH=/home/cody/obj/y/tmp/work/x86_64-poky-linux-gnueabi/rust-cross-arm/git+gitAUTOINC+c654a07d29-r0/build/x86_64-linux/stage2/lib/arm-poky-linux-gnueabi:$LD_LIBRARY_PATH   x86_64-linux/stage2/bin/arm-poky-linux-gnueabi/rustc --cfg stage2  -O --cfg rtopt --cfg ndebug -C rpath  --target=x86_64-linux   -C prefer-dynamic -D warnings -L "x86_64-linux/rt" -L "/home/cody/obj/y/tmp/work/x86_64-poky-linux-gnueabi/rust-cross-arm/git+gitAUTOINC+c654a07d29-r0/build/x86_64-linux/llvm/Release/lib"    --out-dir x86_64-linux/stage2/lib/arm-poky-linux-gnueabi/rustlib/x86_64-linux/lib -C extra-filename=-d8ace771 /home/cody/obj/y/tmp/work/x86_64-poky-linux-gnueabi/rust-cross-arm/git+gitAUTOINC+c654a07d29-r0/git/src/librustc_llvm/lib.rs
| error: could not exec the linker `arm-poky-linux-gnueabi-`: No such file or directory (os error 2)
```

Essentially, 'LINK_foo' is empty.

Heres a list:

```
# grep -L '^LINK_.*' -r cfg/mk
./x86_64-unknown-bitrig.mk
./powerpc-unknown-linux-gnu.mk
./i386-apple-ios.mk
./arm-unknown-linux-gnueabi.mk
./x86_64-unknown-freebsd.mk
./i686-pc-windows-gnu.mk
./mips-unknown-linux-gnu.mk
./armv7s-apple-ios.mk
./x86_64-unknown-dragonfly.mk
./arm-unknown-linux-gnueabihf.mk
./x86_64-unknown-openbsd.mk
./mipsel-unknown-linux-gnu.mk
./x86_64-unknown-linux-musl.mk
./x86_64-apple-ios.mk
./armv7-apple-ios.mk
```

**Comment by jmesmon** [comment]
@nyx I think this is your issue.

**Comment by jmesmon** [comment]
Dup of https://github.com/rust-lang/rust/issues/25723 , apparently.

**Comment by alexcrichton** [comment]
I believe this will be fixed by #25799

**Comment by jmesmon** [comment]
Closing as a duplicate.
